src/oss.yml

```diff
@@ -24,6 +24,8 @@ paths:
     $ref: "./common/paths/authorizations.yml"
   /authorizations/{authID}:
     $ref: "./common/paths/authorizations_authID.yml"
```

Comment:
The `/authorizations` routes above here are for v2 auth. We need to add copies with `servers` set to also cover the v1 endpoints.